tool/jt.rb

```diff
@@ -446,6 +446,9 @@ def help
       jt test specs fast                             run all specs except sub-processes, GC, sleep, ...
       jt test spec/ruby/language                     run specs in this directory
       jt test spec/ruby/language/while_spec.rb       run specs in this file
+      jt test test/mri/tests/test_find.rb [-- <MRI runner options>]
```

Comment:
I always use `jt test mri test/mri/tests/test_find.rb` for clarity of what is being run.
I think it would be clearer in this help too.

Reply:
done